timechart/displays/main_display.py

```diff
@@ -430,6 +430,14 @@ def __init__(self, parent=None, args=[], macros=None, show_pv_add_panel=True, co
         self.time_span_limit_seconds = None
         self.data_sampling_mode = ASYNC_DATA_SAMPLING
 
+        parser = argparse.ArgumentParser(description='TimeChart')
```

Comment:
Hmm, I think this should have gone in `timechart_launcher/main.py` - it has an arg parser already. Generally, you want to keep this sort of thing in only one place.

After that, in the launcher, we'd do something like:
```python
for pv in args.pvs:
   display.add_pv(pv)
```
where `add_pv` then becomes a reusable function. We could probably re-implement `add_curve` while we're at it to rely on the new `add_pv`, even...